File: src/python/examples/create_project.py
from halchemy import Api


def main():
    api = Api('http://localhost:2112')
    root = api.get()
    
    project = {
        'name': 'Sample project',
        'description': 'A project to create sample tasks for'
    }
    
    api.post_to_rel(root, 'projects', project)
    


if __name__ == '__main__':
    main()

File: src/node/halchemy/examples/js/createProject.js
const halchemy = require('halchemy')


async function main() {
    const api = new halchemy.Api('http://localhost:2112')
    const root = await api.get()
    
    const project = {
        name: 'Sample project',
        description: 'A project to create sample tasks for'
    }
    
    const relSpec = {
        resource: root,
        rel: 'projects'
    }
    await api.postToRel(relSpec, project)
}

main()


